### ISSUE TITLE
setGraphPerms method not working

### ISSUE THREAD

**Issue by srinathgit** [marklogic/marklogic-sesame#123]
I have a code snippet that creates a graph and sets a role "read-privileged' with 'READ" capability. When I check the permission of the graph, I see the set permission missing. I see only the standard 'rest-reader' and 'rest-writer' roles assigned to the document. 

    @Test
    public void testGraphPerms1()
            throws Exception {
        GraphManager gmgr = databaseClient.newGraphManager();
        Resource context = testAdminCon.getValueFactory().createURI("http://marklogic.com/test/graph/permstest");

        String defGraphQuery = "CREATE GRAPH <http://marklogic.com/test/graph/permstest> ";
        MarkLogicUpdateQuery updateQuery = testAdminCon.prepareUpdate(QueryLanguage.SPARQL, defGraphQuery);
        updateQuery.setGraphPerms(gmgr.permission("read-privileged", Capability.READ));
        updateQuery.execute();
}

Permissions of the document:
 &lt;sec:permission xmlns:sec="http://marklogic.com/xdmp/security">
 &lt;sec:capability&gt;update &lt;/sec:capability>
 &lt;sec:role-id&gt;15520654661378671735 &lt;/sec:role-id>
 &lt;/sec:permission&gt;
 &lt;sec:permission xmlns:sec="http://marklogic.com/xdmp/security">
 &lt;sec:capability&gt;read &lt;/sec:capability>
 &lt;sec:role-id&gt;7089338530631756591 &lt;/sec:role-id>
 &lt;/sec:permission&gt;

**Comment by xquery** [comment]
I see the issue, I've added fix to PR #141 though you need to amend your test to set to a pre-existing role

this is illustrated in MarkLogicGraphPerms.java with;

```
    @Test
    public void testUpdateQueryWithPerms()
            throws Exception {

        GraphManager gmgr = adminClient.newGraphManager();
        Resource context = conn.getValueFactory().createURI("http://marklogic.com/test/graph/permstest");

        String defGraphQuery = "INSERT DATA { GRAPH <http://marklogic.com/test/graph/permstest> { <http://marklogic.com/test> <pp1> <oo1> } }";
        String checkQuery = "ASK WHERE {  GRAPH <http://marklogic.com/test/graph/permstest> {<http://marklogic.com/test> <pp1> <oo1> }}";
        MarkLogicUpdateQuery updateQuery = conn.prepareUpdate(QueryLanguage.SPARQL, defGraphQuery);
        updateQuery.setGraphPerms(gmgr.permission("test-role", Capability.READ));
        updateQuery.execute();

        BooleanQuery booleanQuery = conn.prepareBooleanQuery(QueryLanguage.SPARQL, checkQuery);
        boolean results = booleanQuery.evaluate();
        Assert.assertEquals(true, results);

        conn.clear(context);
    }
```

once the PR gets reviewed and merged will send your way.

**Comment by srinathgit** [comment]
Verified that setGraphPerms  work and permissions are set  on the triple document.
